release-notes.txt

```diff
@@ -1,3 +1,521 @@
+      consumption
+    * stm32l1/vendor: update vendor files to v2.3.0
+    * arduino/sketches: build sketches as a module
+    * pic32-wifire: add support for flashing with pic32prog
+
+CPU
+---
+    + cpu/esp32: lwIP netdev
+    * cpu/esp*: various improvements
+    * cpu/kinetis: enable HWRNG for k64f
+    - cpu/mips32r2_generic: remove cpu.
+    * cpu: Moved stdio_init() prior to periph_init() for ARM targets
+    * cpu/nrf5x: decelare radio_nrfble feat for all nrfs
+    + cpu/nrf5x: implement wdt peripheral driver
+    + cpu/sam0_common: add hwrng driver
+    * cpu/sam0_common/gpio: don't hard-code number of ports
+    * cpu/sam0: update doc.txt with new MCU families
+    * cpu/saml21: Make Low-Power SRAM available to programs
+    + cpu/stm32_common: add watchdog for stm32
+    * cpu/stm32f(2|4|7): add riotboot requirements
+    * cpu/stm32l1: optimize power consumption
+    * sam0: allow flashing with JLinkExe
+    + sam0: Implement watchdog driver
+    * cpu/samr30: update vendor files using ASF 3.35.1
+    + cpu/efm32: provide periph_uart_mode
+
+Device Drivers
+--------------
+    + drivers: add mtd wrapper for periph_flashpage
+    + drivers/cc110x: Complete rewrite from scratch, and detailed documentation
+    * drivers/include/periph/eeprom: Changed uint8_t* to void* in API
+    * drivers/periph/i2c: Updated i2c_release() to return void
+    + drivers/ph_oem: support for Atlas Scientific pH OEM sensor
+    + drivers: support for NXP PCA9685 I2C 16-channel, 12-bit PWM controller
+    + usbus: Add CDC-ACM (Serial console) function
+    * periph/timer: Fix return codes of timer API
+    + can: add ncv7356 SW transceiver driver
+    + drivers/lis2dh12: add I2C mode
+    + drivers/periph: add doc on power management aspects
+    * drivers/mrf24j40: add pseudomodules for MRF24J40MA/B/C/D/E
+    * riotboot: define if building the bootloader
+
+Build System / Tooling
+----------------------
+    * dist/tools/pyterm: drop loglevel from output
+    * make: docker: delegate bind mounts to the containers
+    * Makefile.base: implement relative path linking without 'realpath'
+    * Makefile.base: use thin static archives.
+    * Makefile.include: add flash-only as dependency of term
+    * Makefile.include: do not build HEXFILE by default anymore
+    * Makefile.include: require make version 4.
+    * makefiles: add support for sysfs gpio debug adapter
+    * Makefiles: add support to generate both `.hex` and `.bin` file and add
+      FLASHFILE variable
+    * makefiles/docker.inc.mk: various improvements
+    * makefiles/murdock.inc.mk: change policy to run tests by default
+    * make: introduce $(CLEAN)
+    + murdock: introduce 'TEST_ON_CI_BLACKLIST'
+    * toolchain/cflags: enable dwarf compression (save 50% HDD)
+    * edbg: verify before flash
+    + ci: add codespell check script for tracking typos
+    + Makefile.include: add cleanterm target and use it for tests
```

Comment:
Should we mention the stale bot in here?

Reply:
Fixed in a54cb60. All hail the stale bot!